proxy/src/proxy.rs

```diff
@@ -79,7 +79,7 @@ static NUM_BYTES_PROXIED_COUNTER: Lazy<IntCounterVec> = Lazy::new(|| {
     register_int_counter_vec!(
         "proxy_io_bytes_per_client",
```

Comment:
Should probably rename this
```suggestion
        "proxy_io_bytes",
```